server/routes/items.js

```diff
@@ -110,13 +117,19 @@ router.post(
       });
 
       ownerNames = identityResponse.data.accounts[0].owners[0].names;
+      const fullName = ownerNames[0].split(' ');
+      firstName = fullName[0];
+      lastName = fullName[fullName.length - 1];
+
       if (!isAuth) {
         balances = identityResponse.data.accounts[0].balances;
       }
     }
     // processorToken is only set if IS_PROCESSOR is true in .env file and
     // therefore isAuth is false
     let processorToken = null;
+    let custUrl = null;
```

Comment:
took me a while to realize this referred to a customer URL, maybe make that more explicit? `customerUrl`

Reply:
will do. thanks.